CommonTools/PileupAlgos/python/customizePuppiTune_cff.py

```diff
@@ -37,6 +37,10 @@ def UpdatePuppiTuneV15(process, runOnMC=True):
   process.puppiNoLep.PtMaxNeutralsStartSlope = 20.
   process.puppiNoLep.NumOfPUVtxsForCharged = 2
 
+  from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+  phase2_common.toModify( process.puppi, EtaMinUseDeltaZ = 4.0)
```

Comment:
isn't this together with L36 redundant with CommonTools/PileupAlgos/python/Puppi_cff.py?
it's rather confusing that there is a customization that doesn't customize anything in the end.

Reply:
the customization is just here such that it can be reviewed before backporting. In principle the whole file can be dropped from the master.

Reply:
if the tune is equal to what is already in the default/standard configuration, it would then be better to drop the tune or make it explicitly a no-op.
I was thinking of a possible use case where the old tunes would still be useful. In that case, this file could be a good place to hold them (V14 etc)

The current setup looks more like a way to increase the maintenance burden for essentially zero benefit.